doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -49,14 +49,15 @@ This chapter is organized as follows:
     - VIM Core services (keystone, cinder, nova, neutron etc.)
     - Tenant Separation
     - Host aggregates providing resource pooling
-    - Flavor management
+    - Flavor* management
 *	Underlying Resources: are what provides the resources that allow the Consumable Infrastructure Resources and Services to be created and managed by the Cloud Infrastructure Management Software (VIM).
     - Virtualisation
     - Physical infrastructure
       -	Compute
       -	Network: Spine/Leaf; East/West and North/South traffic
       -	Storage
 
+> * Please note "flavours" is used in the Reference Model and shall continue to be used in the context of specifing the geometry of the virtual resources. The term "flavor" will be used in the OpenStack and this document context including when specifying configurations; the OpenStack term flavor includes the profile configuration information as "extra specs".
```

Comment:
Spelling - should be flavors

Reply:
@sukhdevkapur RM uses "flavours" for sizing and profiles for "host configurations". OpenStack, as you know, doesn't have host configurations but uses flavors for both sizing and in "extra-specs" to specify the host configuration desired by workloads.

RA1 refers to both RM and OSTK and the Note just points out these term differences.

Reply:
@sukhdevkapur There are now only two occurrences of "flavour" - 1. In the Note as to the difference between RM flavour and RA1 OSTK flavor, and another where there is again a reference to RM. 

We could delete the Note and edit the second use such that it doesn't have to use "flavour".

Reply:
@pgoyal01 Could you please indicate from where "flavours" is coming from - the source? (it is being created as a "surrogate" of OpenStack Flavors, so its specific to Anuket)

Reply:
@joaoarodrigues No "flavours" are not a surrogate for OpenStack flavors. The note states clearly that "flavours" is a term used in the RM. The note also makes clear that OpenStack flavor is also used to specify configuration and not only sizing; in RM configurations are specified by "profiles."

Reply:
In previous versions of RM, we had a table that specified various flavours -- tiny, small, large, etc. - and the vCPU, RAM, disk, etc. sizing. The flavours were used with the profiles (such as Basic). We have deleted the pre-defined flavours from RM and that may be the cause of some of the confusion.